tests/integration/models/os.spec.ts

```diff
@@ -5,23 +5,33 @@ import * as _ from 'lodash';
 import * as m from 'mochainon';
 import {
 	balena,
-	BalenaSdk,
 	credentials,
 	givenAnApplication,
 	givenLoggedInUser,
 	IS_BROWSER,
 } from '../setup';
+import BalenaSdk = require('../../..');
 const { expect } = m.chai;
 
 const eventuallyExpectProperty = <T>(promise: Promise<T>, prop: string) =>
 	expect(promise).to.eventually.have.property(prop);
 
+const {
+	_getDeviceTypes,
+	_getOsVersions,
+	_getDownloadSize,
+	_getMaxSatisfyingVersion,
+	_clearDeviceTypesEndpointCaches,
+} = balena.models.os as ReturnType<
```

Comment:
I would prefer `const osModel = balena.models.os as ... ` to be clear where these come from, but I'm fine with it.